packages/core/src/components/steps-modal/steps-modal.spec.ts

```diff
@@ -0,0 +1,211 @@
+import { newSpecPage, SpecPage } from '@stencil/core/testing'
+
+import { AtomStepsModal } from './steps-modal'
+
+describe('atom-steps-modal', () => {
+  let page: SpecPage
+
+  beforeEach(async () => {
+    page = await newSpecPage({
+      components: [AtomStepsModal],
+      html: `
+        <atom-button id="open-modal-steps">Open Modal</atom-button>
+        <atom-steps-modal
+            steps="3"
+            trigger="open-modal-steps"
+            steps-titles="Step 1, Step 2, Step 3"
+        >
+        <div slot="step-1">Step 1 Content</div>
+        <div slot="step-2">Step 2 Content</div>
+        <div slot="step-3">Step 3 Content</div>
+      </atom-steps-modal>`,
+    })
+  })
+
+  it('should render modal with default values', async () => {
+    expect(page.root).toEqualHtml(`
+        <atom-steps-modal
+        steps="3"
+        trigger="open-modal-steps"
+        steps-titles="Step 1, Step 2, Step 3"
+        >
+        <atom-modal
+            trigger="open-modal-steps"
+            alert-type=""
+            has-divider="false"
+            primary-text="Primary"
+            secondary-text="back"
```

Comment:
block: in storybook the text of the buttons aren't good, it is better add another example, like `primary-text="Next"` and `secondary-text="Cancel"`, `back` for example is in lowercase

<img width="593" alt="image" src="https://github.com/user-attachments/assets/4a714b98-5d3d-4235-b0ba-eb9a804ccb06">